cypress/integration/pages/onDemandAudio/tests.js

```diff
@@ -11,6 +11,20 @@ import getDataUrl from '../../../support/helpers/getDataUrl';
 
 export default ({ service, pageType, variant, isAmp }) => {
   describe(`Tests for ${service} ${pageType}`, () => {
+    describe('Brand image visible above 400px, not visible below 400px', () => {
+      it(`Should display image on default viewport (1000x660))`, () => {
+        cy.get('div[data-e2e="on-demand-image"]').find('img');
+      });
+
+      it(`Should not display image on iphone-6 screen (375x667)`, () => {
+        cy.viewport('iphone-6');
+
+        cy.get('div[data-e2e="on-demand-image"]')
+          .find('img')
+          .should('not.be.visible');
+      });
+    });
```

Comment:
Any idea why this test is on this branch? It should probably be moved to a seperate PR?

Reply:
As mentioned on Slack, this should be removed from this PR and doesn't need to be put on another one.